flood_adapt/object_model/direct_impacts.py

```diff
@@ -442,6 +477,9 @@ def _create_infometrics(self, fiat_results_df) -> Path:
         else:
             ext = ""
 
+        if self.hazard.event_mode == "risk":
```

Comment:
should this be part of the create_infometrics method? I see this as a different process so maybe it should be in the general post-process method?